web/src/features/panels/zone/EstimationCard.tsx

```diff
@@ -105,12 +105,14 @@ export default function EstimationCard({
   }
   const selectedData = data?.datetimes[selectedDatetimeString]?.z[zoneKey];
 
-  const estimationMethod = selectedData?.em;
+  const estimationMethod = selectedData?.em || undefined;
 
-  if (!estimationMethod) {
+  const isAggregated = !isHourly;
+
+  if (!estimationMethod && !isAggregated) {
```

Comment:
This now shows the estimation card on aggregated data. It wasn't showing before.

now:
<img width="376" alt="image" src="https://github.com/user-attachments/assets/4aaa76da-c146-43e8-9f1e-1350bc3f671b" />